### ISSUE TITLE
Choice of max_depth prameter

### ISSUE THREAD

**Issue by prismformore** [zhyever/Monocular-Depth-Estimation-Toolbox#38]
May I know what is the criterion of choosing max_depth? I see that you use 200 for cityscapes and 10 for nyu. Thank you!

**Comment by prismformore** [comment]
I see that in this [repo](https://github.com/sjsu-smart-lab/Self-supervised-Monocular-Trained-Depth-Estimation-using-Self-attention-and-Discrete-Disparity-Volum) they use 80 for cityscapes.

**Comment by zhyever** [comment]
Till now, I have not compared model performance on cityscapes. The config in cs now is not much value for reference. 

It seems I did a statistic evaluation about the depth range of cityscape and found the max value could be 200m. So I simply set `max_depth` to 200. The reason why your provided repo sets this max value to 80m can be that the authors wanna align the evaluation metric with KITTI dataset. You know, KITTI depth has a range of 0-80m.

Remember, max_depth in dataset configs is only used for model evaluation. It filters distant pixels over the max_depth value. For most papers for depth estimation, it is fairer to choose a similar max_depth for comparison. Hence, if you wanna compare with the method you provide, just set the max_depth to 80 for fair comparison! Generally, 10m for nyu and 80m for kitti. I'm not sure about the cityscapes. Hope these are helpful for you.

**Comment by zhyever** [comment]
Till now, I have not compared model performance on cityscapes. The config in cs now is not much value for reference. 

It seems I did a statistic evaluation about the depth range of cityscape and found the max value could be 200m. So I simply set `max_depth` to 200. The reason why your provided repo sets this max value to 80m can be that the authors wanna align the evaluation metric with KITTI dataset. You know, KITTI depth has a range of 0-80m.

Remember, max_depth in dataset configs is only used for model evaluation. It filters distant pixels over the max_depth value. For most papers for depth estimation, it is fairer to choose a similar max_depth for comparison. Hence, if you wanna compare with the method you provide, just set the max_depth to 80 for fair comparison! Generally, 10m for nyu and 80m for kitti. I'm not sure about the cityscapes. Hope these are helpful for you.
